app/experimenter/nimbus-ui/src/components/PageRequestReview/FormLaunchDraftToPreview.tsx

```diff
@@ -0,0 +1,60 @@
+/* istanbul ignore file until EXP-866 final */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import Alert from "react-bootstrap/Alert";
+import Form from "react-bootstrap/Form";
+import { ReactComponent as ExternalIcon } from "../../images/external.svg";
+import { EXTERNAL_URLS } from "../../lib/constants";
+import LinkExternal from "../LinkExternal";
+
+const FormLaunchDraftToPreview = ({
+  isLoading,
+  onSubmit,
+  onLaunchWithoutPreview,
+}: {
+  isLoading: boolean;
+  onSubmit: () => void;
+  onLaunchWithoutPreview: () => void;
+}) => {
+  return (
+    <Alert variant="warning">
+      <Form className="text-body">
+        <p>
+          Do you want to test this experiment before launching to production?{" "}
+          <LinkExternal href={EXTERNAL_URLS.PREVIEW_LAUNCH_DOC}>
+            <span className="mr-1">Learn more</span>
+            <ExternalIcon />
```

Comment:
Proactively filed https://jira.mozilla.com/browse/EXP-874 since this is starting to become a common thing

Reply:
@AnaMedinac should we have a tooltip here? I know there's a "Learn More" link and maybe that's sufficient, but could help to further disambiguate the meaning of "Preview" to have a one or two sentence summary in the tooltip.